creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -108,16 +123,18 @@ module.exports = async function ({ logger, syncType, syncRequestParameters }) {
     `------------------Process SYNC | ${logMsgString} | Primary clock value ${primaryClockValue}------------------`
   )
 
-  // Issue sync request to secondary
+  // Issue sync request to secondary with forceResync = true
```

Comment:
why do we want to force a resync for every sync? is this supposed to only happen when the syncMode is `MergePrimaryAndSecondary`?

Reply:
yep, just incomplete